charts/landscaper/templates/cluster-kubeconfig-secret.yaml

```diff
@@ -0,0 +1,26 @@
+# SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
The landscaper helm chart currently also contains rbac rules for creating crd's, etc.. that are needed on the remote cluster.
But with that change these rbac rules and service account would get deployed on the host cluster which is not really what we want.

So I suggest to not deploy the rbac rules and service accounts when a remote landscaper cluster is specified.
or maybe move the definition to a dedicated chart as it is done in the gardener with their `runtime` and `application` charts

Reply:
done